samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -78,6 +78,9 @@ static uint32_t led_state;
 
 static struct lwm2m_ctx client;
 
+/* Array with supported PULL firmware update protocols */
+static uint8_t supported_protocol[1] = {0};
```

Comment:
Minor nit, right hand side of `=` is not really needed as the array is zero anyway (because it is static)